deploy/overlays/default/trace-context-injector-args.yaml

```diff
@@ -0,0 +1,14 @@
+# This patch sets arguments for the trace context injector
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: trace-context-injector-webhook-deployment
+spec:
+  template:
+    spec:
+      containers:
+      - name: trace-context-injector
+        args:
+          - -rate=1.0                 
```

Comment:
does this patch actually do anything?  It looks like it just uses the defaults...  A better example patch might be showing how you can decrease the sampling rate to 10% or something.

Reply:
done